static/app/components/feedback/useFeedbackOnboarding.tsx

```diff
@@ -45,3 +46,10 @@ export function useHaveSelectedProjectsSetupFeedback() {
     fetching,
   };
 }
+
+export function useFeedbackFromSlack() {
+  const {getParamValue: getReferrer} = useUrlParams('referrer', '');
+  const isFromSlack = getReferrer() === 'slack';
```

Comment:
what if its from an email?

Reply:
we can also apply this to emails. do you know the referrer name for that?

Reply:
actually, nvm i can figure it out from the emails i've gotten!